readme.md

```diff
@@ -211,6 +221,8 @@ Then change the font to one of Powerline options. 13pt Cousine is a good choice.
 - [iterm2] - A terminal emulator and replacement for Mac's default Terminal app.
 - [Screenhero] - Coding oriented screen share with multi mouse/cursor.
 - [Slack] - A messaging app (corporate IRC 👔).
+- [Jumpcut] - Keep a record of things that you have copied (clipboard buffering).
+- [Spectacle] - A free alternate to Moom to automate your screen re-sizing.
```

Comment:
You should throw these in an Issue. I wont put spectacle in there cuz I have moom, but something I might toss in an "Alternates" portion of the readme in the future. Jumpcut I would include, but don't want to commit it to the readme till I actually toss it in.